src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -163,51 +175,55 @@ export default class GitOpsConfiguration extends Component<GitOpsProps, GitOpsSt
                                 </label>
                             </span>
                         </label>
                     </div>
                     <div>
                         <label className="tertiary-tab__radio">
-                            <input type="radio" name="status" value={GitProvider.Github} checked={this.state.tab === "github"} onChange={this.handleGitopsTab} />
+                            <input type="radio" name="status" value={GitProvider.GitLab} checked={this.state.tab === "gitlab"} onChange={this.handleGitopsTab} />
                             <span className="tertiary-tab sso-icons">
-                                <aside className="login__icon-alignment"><GitHub /></aside>
-                                <aside className="login__text-alignment"> GitHub</aside>
+                                <aside className="login__icon-alignment"><GitLab /></aside>
+                                <aside className="login__text-alignment"> GitLab</aside>
                                 <label>
-                                    {this.state.lastActiveGitOp?.provider?.toLocaleLowerCase() == "github" ? <aside className="login__check-icon"><img src={Check} /></aside> : ""}
+                                    {this.state.lastActiveGitOp?.provider?.toLocaleLowerCase() == "gitlab" ? <aside className="login__check-icon"><img src={Check} /></aside> : ""}
                                 </label>
                             </span>
                         </label>
                     </div>
                 </div>
                 <div className="flex column left top pl-20">
                     <div className="gitops__id fw-5 fs-13 mb-8">Git Host*</div>
-                    <input value={this.state.form.host} type="text" name="githost" className="form__input"
+                    <input autoComplete="off" value={this.state.form.host} type="text" name="githost" className="form__input"
                         onChange={(event) => this.handleChange(event, 'host')} />
                 </div>
                 <div className="flex column left top pt-16 pl-20 pb-6">
                     <div className="gitops__id fw-5 fs-13 mb-8">
-                        {this.state.tab === GitProvider.Github ? "GitHub Organisation ID" : "GitLab Group ID"}
+                        {this.state.tab === GitProvider.Github ? "GitHub Organisation ID*" : "GitLab Group ID*"}
                     </div>
-                    <input value={this.state.form[key]} type="text" name="gitorg" className="form__input"
+                    <input autoComplete="off" value={this.state.form[key]} type="text" name="gitorg" className="form__input"
                         onChange={(event) => { this.handleChange(event, key); }} />
+                    {this.state.isError.gitHubOrgId && <div className="form__error">
```

Comment:
It should be `githubOrgId` in case on github and `gitlabOrgId` in case of gitlab. Handle it using variable key, it declared at the top of render()